code/modules/clothing/gloves/color.dm

```diff
@@ -8,6 +8,11 @@
 	item_color="yellow"
 	burn_state = -1 //Won't burn in fires
 
+/obj/item/clothing/gloves/color/yellow/syndicate //Actually red, this is what you get when you just arbitrarily decide that one color is better than the rest!

```

Comment:
I admitted I was shitpostin last message you won you won congratulations

:tada: :tada: :cop:  :clap: :clap: